gen3utils/deployment_changes/generate_comment.py

```diff
@@ -376,7 +381,10 @@ def check_services_on_branch(versions_block):
     services_on_branch = []
     for service in versions_block:
         version = versions_block.get(service)
-        if "quay.io/cdis" not in version or len(version.split(":")) < 2:
+        if (
+            "quay.io/cdis" not in version
+            and "dkr.ecr.us-east-1.amazonaws.com" not in version
```

Comment:
@MaribelleHGomez when we started using ECR instead of Quay, the other functions were updated but not this one, so ECR images were always ignored and we didn't check if they were pinned to a branch